yarpc-encoding-raw/src/main/java/io/yarpc/encoding/raw/OnewayRawAdapter.java

```
@@ -0,0 +1,35 @@
+package io.yarpc.encoding.raw;
+
+import io.yarpc.core.Context;
+import io.yarpc.core.transport.OnewayHandler;
+import io.yarpc.core.transport.TransportException;
+import io.yarpc.core.transport.TransportRequest;
+import java.io.IOException;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public class OnewayRawAdapter implements OnewayHandler {
+  @NonNull OnewayRawHandler inbound;
+
+  @Override
+  public void handleOneway(TransportRequest req) throws TransportException {
+    //TODO: Change the default context and extract context from request.
+    Context ctx = new Context();
+    try {
+      int bytesToRead = req.getBody().available();
+      int totalBytes = 0;
+      while (bytesToRead > 0) {
+        byte[] chunk = new byte[req.getBody().available()];
+        bytesToRead = req.getBody().read(chunk);
+        totalBytes += bytesToRead;
+      }
+
+      byte[] input = new byte[totalBytes];
+      req.getBody().read(input);
+      inbound.handleOneway(ctx, input);
+    } catch (IOException e) {
+      e.printStackTrace();
```

For now, throw this as `new RuntimeException(e)` instead of printing the stack trace. We don't want to accidentally lose exceptions.